src/test/java/uk/gov/ons/ssdc/supporttool/testhelper/IntegrationTestHelper.java

```diff
@@ -71,12 +94,40 @@ public void testGet(
     }
   }
 
+  public void testPost(
+      int port,
+      UserGroupAuthorisedActivityType activity,
+      BundleUrlGetter bundleUrlGetter,
+      BundlePostObjectGetter bundlePostObjectGetter) {
+    setUpTestUserPermission(activity);
+    BundleOfUsefulTestStuff bundle = getTestBundle();
+
+    String url = String.format("http://localhost:%d/api/%s", port, bundleUrlGetter.getUrl(bundle));
+    Object objectToPost = bundlePostObjectGetter.getObject(bundle);
+    ResponseEntity<String> response = restTemplate.postForEntity(url, objectToPost, String.class);
+    assertThat(response.getStatusCodeValue()).isBetween(200, 299);
```

Comment:
I would consider replacing this with something like:   assertThat(response.getStatusCodeValue()).as("Testing POST").isEqualTo(201);
Gives some information regarding what has failed,  and all of them pass with 201.  No need for a wide match?  could be overloaded/parameterised in the future if required.